v2.0.21 build/make warning

hi, i got the following warning (yeah i know, it's just a warning) when building release 2.0.21 on ubuntu trusty (14.04LTS):
inspircd/src/modules/m_hideoper.cpp: In member function ‘virtual ModResult ModuleHideOper::OnNumeric(User*, unsigned int, const string&)’:
inspircd/src/modules/m_hideoper.cpp:97:88: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 5 has type ‘size_t {aka unsigned int}’ [-Wformat=]
user->WriteNumeric(252, "%s %lu :operator(s) online", user->nick.c_str(),  opercount);

--- a/src/modules/m_hideoper.cpp
+++ b/src/modules/m_hideoper.cpp
@@ -94,7 +94,7 @@ class ModuleHideOper : public Module
 		if (opercount)
 		{
 			active = true;
-			user->WriteNumeric(252, "%s %lu :operator(s) online", user->nick.c_str(),  opercount);
+			user->WriteNumeric(252, "%s %lu :operator(s) online", user->nick.c_str(),  static_cast<unsigned long>(opercount));
 			active = false;
 		}
 		return MOD_RES_DENY;

